kytos/core/napps/manager.py

```diff
@@ -90,7 +89,6 @@ def install(self, napp_uri, enable=True):
             dst = self._installed / napp.username / napp.name
             self._create_module(dst.parent)
             shutil.move(str(napp_folder), str(dst))
-            os.chmod(dst, mode=0o755)
```

Comment:
@erickvermot For now, I think that do not set this will cause a problem in some cases because of strange users' umask.

But we can leave with that for now, and wait until this problem happen again. I will accept this as it is now.